### PR TITLE
Oliver long term load tests

### PR DESCRIPTION
**Pull request by olliejth**
Created tests for load.py and renamed load to load_long.py

### CODE REVIEW

**Review by mehrdadhalali — APPROVED** [review]

File: long_term/test_load_long.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+        mock_to_csv.assert_called_once_with(res, index=False)
+
+    @patch("pandas.DataFrame.to_csv")
+    def test_create_csv_error_on_invlaid_input(self, mock_to_csv):
+
+        mock_df = "string"
+
+        with pytest.raises(TypeError) as err:
+            create_csv(mock_df)
```
Comment (Ruyzambrano) [inline long_term/test_load_long.py]: Why do you need to define mock_df here? Can't you just pass a string into the function?

File: long_term/test_load_long.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+class TestGetS3Client():
+
+    @patch("load_long.client")
+    def test_get_s3_client_core_functionality(self, mock_s3_client):
+        test_env = {
+            "AWS_rvbyaulf_KEY": "test_access_key",
+            "AWS_rvbyaulf_SECRET_KEY": "test_secret_key"
+        }
+
+        with patch.dict('load_long.ENV', test_env):
```
Comment (YakMan101) [inline long_term/test_load_long.py]: Change this to a decorator for consistency

File: long_term/test_load_long.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# pylint: skip-file
+
+"""Test file for load functions."""
+
+from unittest.mock import patch, MagicMock
```
Comment (YakMan101) [inline long_term/test_load_long.py:5]: Add line between native modules, external modules and functions imported from local files.

**Review by YakMan101 — CHANGES_REQUESTED** [review]
Added some comments, could you change these first.

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**